### PR TITLE
Adapt Packit configuration to a newly branched Fedora

### PR DESCRIPTION
**Pull request by jkonecny12**
Add daily builds for f34-devel branch.

### CODE REVIEW

**Comment by jkonecny12** [comment]
Packit won't work until COPR will support Fedora 34 chroot.

**Comment by jkonecny12** [comment]
Related https://github.com/packit/packit-service/issues/973 it's not blocking this PR but explains why you don't see the service here.

**Review by poncovka — APPROVED** [review]
Looks good to me. Thanks!

**Comment by jkonecny12** [comment]
COPR now supports F34.

**Review by VladimirSlavik — APPROVED** [review]
Looks good to me. Thank you!

**Comment by jkonecny12** [comment]
Kickstart tests are not yet adapted to after branching. Have to skip this check.

File: .packit.yml [hunk 1]
```diff
@@ -13,14 +13,19 @@ jobs:
     trigger: pull_request
     metadata:
       targets:
-        - fedora-rawhide
+        - fedora-34
```
Comment (lachmanfrantisek) [inline .packit.yml:16]: Would `fedora-development` alias ([docs](https://packit.dev/docs/configuration/#available-copr-build-targets)) help to not need to change this every 6 months? (Will lead to `rawhide` or `rawhide + branched fedora`.)

Comment (lachmanfrantisek) [reply .packit.yml:16]: OK, I now see you have a dedicated branch for `f34` and my comment does not make much sense then.

Comment (jkonecny12) [reply .packit.yml:16]: Hah, that's great improvement. Thanks for pointing that!

**Pull request merged**